packages/devtools_app/lib/src/debugger/flutter/console.dart

```diff
@@ -68,10 +69,11 @@ class _ConsoleState extends State<Console> {
                     itemExtent: CodeView.rowHeight,
                     controller: scrollController,
                     itemBuilder: (context, index) {
-                      return Text(
-                        lines[index],
+                      return RichText(
+                        text: TextSpan(
+                            children: maybeConvertToAnsiText(
+                                lines[index], textStyle)),
```

Comment:
there are still two parens next to each other here. In general, you have added enough commas when all indents are 2 spaces. If you see indents of 4 spaces it means you need to add more commas. This is an annoying edge case because the dartformatter doesn't know you are writing code within a widget build method where just defaulting to two spaces indent would be better.